src/post/Write/DraftRow.js

```diff
@@ -2,28 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'antd';
 import { FormattedMessage } from 'react-intl';
-import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
-import { deleteDraft } from './editorActions';
+import DeleteDraftModal from './DeleteDraftModal';
 
-@connect(null, { deleteDraft })
 class DraftRow extends React.Component {
   static propTypes = {
     id: PropTypes.string.isRequired,
     data: PropTypes.shape().isRequired,
-    pending: PropTypes.bool,
-    deleteDraft: PropTypes.func,
   };
 
-  static defaultProps = {
-    pending: false,
-    deleteDraft: () => {},
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModalDelete: false,
+    };
+  }
+  showModal = () => {
```

Comment:
Those two functions don't have to have body. It would look cleaner this way:
`X = () => this.setState({ showModalDelete: X });`